src/components/Label/Label.tsx

```diff
@@ -63,6 +69,12 @@ export const Label = ({
                 </span>
             )}
             {children && <span className={`${eccgui}-label__other`}>{children}</span>}
+            {additionalElements && (
+                <>
+                    <Spacing vertical size="tiny" />
+                    {additionalElements}
+                </>
+            )}
         </>
     );
 
```

Comment:
in the return statement the test need to be extended for `|| !!additionalElements`, otherwise the label would not be displayed if it only has `additionalElements` given.